packages/scandipwa/src/component/MyAccountMyOrders/MyAccountMyOrders.style.scss

```diff
@@ -26,7 +26,7 @@
         }
     }
 
-    &-Table:not(.MyDownloadable) {
+    &-Table {
         width: 100%;
```

Comment:
Such changes are incomplete. Referencing [the pending PR](https://github.com/scandipwa/scandipwa/pull/2546)